src/en/controllers-creating.md

```diff
@@ -7,7 +7,7 @@ TODO: Review again soon (created: March 2016)
 Use the `juju bootstrap` command to create a controller (and model) for a given
 cloud:
 
-`juju bootstrap [options] <controller name> <cloud name>`
+`juju bootstrap [options] <cloud name> <controller name>`
 
```

Comment:
This may be a good place to additionally explain the alternative usage:

juju bootstrap [options] <cloud name>

which will assign a name automatically etc etc